markusapi/markusapi.py

```diff
@@ -382,6 +382,11 @@ def remove_file_from_repo(self, assignment_id: int, group_id: int, file_path: st
         params = {'filename': file_path}
         return self.submit_request(params, path, 'DELETE')
 
+    def remove_folders(self, assignment_id: int, group_id: int, folder_path: str) -> List[str]:
```

Comment:
This should follow the same naming convention as other methods in this file:

`remove_folder_from_repo`